src/_includes/layout.njk

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{% if title %}{{ title }} – {% endif %}{{ app.title }}</title>
+  <title>{% if title != '' %}{{ title | striptags }} – {% endif %}{{ app.title }}</title>
```

Comment:
I think we can remove the `striptags` filter here since the title will always be a string.